CONTRIBUTING.md

```diff
@@ -85,25 +85,28 @@ development with the following:
 
   For Url:
 
-  `http://localhost:5000/`
+  `http://localhost:5000/auth/myusa/callback`
```

Comment:
Good to know. The issue I was having must have been due to not having the correct scope set before then. I changed both at the same time without checking to see if only changing one fixed the problem. I will update this accordingly.